pylintrc

```diff
@@ -0,0 +1,5 @@
+[TYPECHECK]
```

Comment:
that i don't know - took example from cohorts repo. I will say, it does impact the results of lint.sh .. otherwise i defer to your judgement